msal/src/main/java/com/microsoft/identity/nativeauth/statemachine/results/SignInResult.kt

```diff
@@ -101,3 +117,24 @@ interface SignInResendCodeResult : Result {
         val channel: String,
     ) : SignInResendCodeResult, Result.SuccessResult(nextState = nextState)
 }
+
+// TODO should we call this SignInMFA or just MFA? MFA can only happen during sign in, so it's redundant?
+interface SignInMFARequiredResult: Result {
+    class VerificationRequired(
+        override val nextState: SignInMFARequiredState,
+        val codeLength: Int,
```

Comment:
This implies that the MFA method is a code; which isn't always the case for MFA.